webapp/sources/rudder/rudder-web/src/main/scala/com/normation/plugins/PluginStatus.scala

```diff
@@ -77,9 +78,17 @@ trait PluginStatus {
   /*
    * Is the plugin currently enabled (at the moment of the request) ?
    */
-  def isEnabled(): Boolean = current match {
-    case PluginStatusInfo.EnabledNoLicense | _: PluginStatusInfo.EnabledWithLicense => true
-    case _                                                                          => false
+  def isEnabled(): Boolean = {
+    current match {
+      case PluginStatusInfo.EnabledNoLicense | _: PluginStatusInfo.EnabledWithLicense => true
+      case PluginStatusInfo.Disabled(reason, optInfo)                                 =>
+        val name = optInfo match {
+          case Some(i) => s"'${i.softwareId}' "
+          case None    => ""
+        }
+        ApplicationLoggerPure.Plugin.logEffect.warn(s"Plugin ${name}is disabled: ${reason}")
```

Comment:
this will be outputed each time a limited feature is accessed. It may spam logs, but it seems more important to have the message if number of nodes grow out of bound after start